library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -1303,6 +1314,35 @@ public void initializeOutput(ExtractorOutput output, int trackId, long durationU
       this.output.format(format);
     }
 
+    private static Pair<List<byte[]>, Long> parseFourCCPrivate(ParsableByteArray buffer)
+            throws ParserException {
+        try {
+            buffer.setPosition(buffer.getPosition()+16); // size, skip width(4), height(4),
+                                                         // planes(2), and bitcount(2)
+            long biCompression = buffer.readLittleEndianUnsignedInt();
+            buffer.setPosition(buffer.getPosition()+20); // skip sizeimage(4), xpel/meter(4),
+                                                       // ypel/meter(4), clrUsed(4), clrImportant(4)
+            List<byte[]> privateData = new ArrayList<>();
+            final int baseBytesLeft = buffer.bytesLeft();
+            byte[] scratchBytes = new byte[baseBytesLeft];
+            buffer.readBytes(scratchBytes, 0, baseBytesLeft);
+
+            int realStartPos;
+            for(realStartPos = 0; realStartPos < (baseBytesLeft - 4); ++realStartPos) {
```

Comment:
Is there any documentation about the existence of a reserved byte anywhere (or even some other project skipping it)? If that's what it actually is, we should just skip a byte rather than having a for loop, probably? It would be good to understand exactly what's happening here! Thanks.

NB - Even if the for loop lives, it might be best just to look for 0x00 0x00 0x01? I don't think there's value in checking the 0x0F?